Mathlib/RingTheory/Derivation/Basic.lean

```diff
@@ -439,6 +439,40 @@ theorem leibniz_inv {K : Type*} [Field K] [Module K M] [Algebra R K] (D : Deriva
   · exact D.leibniz_of_mul_eq_one (inv_mul_cancel ha)
 #align derivation.leibniz_inv Derivation.leibniz_inv
 
```

Comment:
```suggestion

section DerivationOfFraction

variable {K : Type*} [Field K] [Module K M] [Algebra R K] (D : Derivation R K M) 
```

Reply:
It is always a balance when to make the variables global vs local in scope. I think a `section` here is appropriate to improve readability. The variables are still declared very close by in the file.

Reply:
Alright, I'll also insert `leibniz_inv` to it, which also uses those same variables